src/plugins/orangehrmAdminPlugin/Api/ModulesAPI.php

```diff
@@ -262,30 +258,6 @@ protected function updateMobileStatus(?bool $enableMobile): void
         //$enableMobile ? $this->createMobileClient() : $this->deleteMobileClient();
```

Comment:
will remove `updateMobileStatus` this method and refactor the class by removing mobile module from the validation rules and request body params